src/Log4Net/NewRelic.LogEnrichers.Log4Net.Tests/Log4NetAppenderTests.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using log4net;
+using log4net.Config;
+using log4net.Core;
+using NewRelic.Api.Agent;
+using NUnit.Framework;
+using Telerik.JustMock;
+
+namespace NewRelic.LogEnrichers.Log4Net.Tests
+{
+	public class Log4NetAppenderTests
+    {
+        [SetUp]
+		public void Setup()
```

Comment:
This setup method is empty. You should either remove it, or simplify some of the tests by moving some of the "arrange" work into the setup method.

Reply:
I thought I fixed this up. Will fix it.